scripts/hid_configurator/configurator.py

```diff
@@ -6,8 +6,9 @@
 import logging
 
 REPORT_ID = 4
-MOUSE_VID = 0x1915
+NORDIC_VID = 0x1915
 MOUSE_PID = 0x52DE
+DONGLE_PID = 0x52DC
```

Comment:
As talked off-line I propose to add device pid in configuration message and check it on the device to filter messages directed to it.